### ISSUE TITLE
Random problem as navigation item

### ISSUE THREAD

**Issue by quantum5** [DMOJ/site#175]
Would be placed under the Problems link. Helps in browsing for a random problem, especially when the current problem is not favoured.

**Comment by quantum5** [comment]
@Xyene @FatalEagle

**Comment by FatalEagle** [comment]
Seems legit. Current position is a bit awkward.

**Comment by Xyene** [comment]
I am strongly opposed to such a change.

This is a problem site, not Wikipedia - while a random button has great value on a wiki where any page's content is at least somewhat accessible, it's less ideal on a site like this because, as you stated, for the average user many problems will not be favoured. Doesn't help that you may get a problem that you've already solved (this ought to be a discussion ticket in its own right).

My logic:
* if you're good and confident that you can solve the majority of problems randomly picked, you've already been on the site long enough (and solved enough problems) so that chances are that you've already solved the problem selected
* if you're not, you'll waste a great deal of time repeatedly requesting random problems

Either way your experience is suboptimal. (side note: dropdowns are still an eyesore, and making eyes sore for the most often-clicked navbar item doesn't sound nice.)

I agree with @FatalEagle that the current position (or maybe just the wording) is awkward, but I do not believe relocating it to a navbar item will fix the issue. If anything, it'll just look worse. Worse than the inconvenience of navigating back (when a problem is not favoured) because it adds noise to the interface for people who couldn't care less about random problems (this should be profiled over a period of a few days/weeks).

**tl;dr:** pls no

**Comment by FatalEagle** [comment]
What about selecting problems based on rating or highest point value solved? Out of all the unsolved problems.

**Comment by quantum5** [comment]
Too much hate
